src/test/system_tests/ceph_s3_tests/test_ceph_s3.js

```diff
@@ -0,0 +1,142 @@
+/* Copyright (C) 2016 NooBaa */
```

Comment:
This file was renamed...
This was my git status before I amended this commit:
![Screen Shot 2023-01-02 at 19 04 53](https://user-images.githubusercontent.com/57721533/210261213-31ae7c12-06c2-47a9-bdd7-843804af3223.png)

Reply:
I will share another trial I had with the code from this branch:
[The first](https://github.com/shirady/learn-action/actions/runs/3836247315/jobs/6530210631) - using sleep after waiting for the backing store to be ready. here is the printing of test pool:
![Screen Shot 2023-01-04 at 10 57 30](https://user-images.githubusercontent.com/57721533/210519173-5e455e99-c692-4b8e-b6ed-4ac504a60756.png)
[The second](https://github.com/shirady/learn-action/actions/runs/3836403533/jobs/6530533780) - without sleep, all the tests are failing!
![Screen Shot 2023-01-04 at 10 58 37](https://user-images.githubusercontent.com/57721533/210519408-a0a51533-d4af-4345-bd00-4c972fa5f0e8.png)
Note: The links are to a temp repo and will not be available after this PR will be closed.